docs/REST API/rest-api.md

```diff
@@ -9242,6 +9218,38 @@ To get the results of an App Store app install, use the [List MDM commands](#lis
  }
 ```
 
+### Download package
+
+> **Experimental feature**. This feature is undergoing rapid improvement, which may result in breaking changes to the API or configuration surface. It is not recommended for use in automated workflows.
+
+_Available in Fleet Premium._
+
+`GET /api/v1/fleet/software/titles/:software_title_id/package?alt=media`
+
+#### Parameters
+
+| Name            | Type    | In   | Description                                      |
+| ----            | ------- | ---- | --------------------------------------------     |
+| software_title_id   | integer | path | **Required**. The ID of the software title to download software package.|
+| team_id | integer | query | **Required**. The team ID. Downloads a software package added to the specified team. |
+| alt             | integer | query | **Required**. If specified and set to "media", downloads the specified software package. |
```

Comment:
What happens if you set it to something else?

Reply:
@rachaelshaw We return this error:

<img width="629" alt="image" src="https://github.com/user-attachments/assets/b049d682-a084-4ac5-88bc-610d39136045">

Reply:
What's the reason for including the `alt` parameter if it's required & only supports one value? Are we planning on changing this to be a "Get or download" endpoint in the future & making `alt` optional?

Reply:
@rachaelshaw We decided in the past that we'll use `alt=media` for download. Ideally, it would be "Get or download", but we don't need "Get" in the UI in this case, so that's why we have `alt` required.